src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const filteredMovies = moviesFromServer.filter(movie => {
+    const lowerCaseQuery = query.toLowerCase().trim();
```

Comment:
[CODE STYLE] - Don't repeat yourself. The `lowerCaseQuery` is calculated for each movie in the array. Instead, calculate it once before the `filter` method.